src/services/import/excel/XLSImporterValidateObservations.ts

```diff
@@ -38,38 +38,21 @@ export default class XLSImporterValidateObservations extends AbstractImporter {
   private age: Repository<Age> = getRepository(Age);
 
   private checkEuRingCodes = async (excelData: DataCheck): Promise<void> => {
-    const statusCached = await this.status
-      .createQueryBuilder('status')
-      .select('status.id')
-      .getRawMany();
-    const speciesMentionedCached = await this.species
-      .createQueryBuilder('species')
-      .select('species.id')
-      .getRawMany();
-    const sexMentionedCached = await this.sex
-      .createQueryBuilder('sex')
-      .select('sex.id')
-      .getRawMany();
-    const ageMentionedCached = await this.age
-      .createQueryBuilder('age')
-      .select('age.id')
-      .getRawMany();
-
     try {
+      const statusCached = await this.status.find({ cache: 60000 });
+      const speciesMentionedCached = await this.species.find({ cache: 60000 });
+      const sexMentionedCached = await this.sex.find({ cache: 60000 });
+      const ageMentionedCached = await this.age.find({ cache: 60000 });
```

Comment:
issue with such cache that it will be stored in postgresql, so it wont give you any boost

Reply:
it's only have some value for hard requests or when instead PG using redis, which currently isnt allowed